packages/solo/test/test-home.js

```diff
@@ -1,16 +1,19 @@
 /* global process */
 
-import '@endo/init/pre-bundle-source.js';
-
 // `test.after.always` does not yet seem compatible with ses-ava
 // See https://github.com/endojs/endo/issues/647
 // TODO restore
-// import { test } from '@agoric/swingset-vat/tools/prepare-test-env-ava';
+// import { test } from '@agoric/swingset-vat/tools/prepare-test-env-ava.js';
 import '@agoric/swingset-vat/tools/prepare-test-env.js';
 // eslint-disable-next-line import/no-extraneous-dependencies
 import test from 'ava';
+// TODO
```

Comment:
Uncomment all the TODOs in this file to reproduce the publishBundle HTTP failure. Otherwise, this should be sufficient to test the HTTP connection type in `@agoric/publish-bundle`, but not sufficient to test for regressions in publishing to a Cosmos chain. I’m not sure where to hook up such a test.